forms.py

```diff
@@ -1,113 +1,78 @@
-from datetime import timedelta
-
-from django.utils import timezone
 from django import forms
+from django.utils import timezone
 
-from plugins.consortial_billing import models
-from plugins.consortial_billing.templatetags.currency import convert_to
+from plugins.consortial_billing import models, logic
 
 
-class Institution(forms.ModelForm):
+class BandForm(forms.ModelForm):
     class Meta:
-        model = models.Institution
-        fields = ('first_name', 'last_name', 'email_address', 'name', 'address', 'postal_code', 'country', 'display')
-
-
-class InstitutionForm(forms.ModelForm):
+        model = models.Band
+        fields = [
+            'country',
+            'currency',
+            'size',
+            'level',
+            'fee',
+        ]
 
     def __init__(self, *args, **kwargs):
-        super(InstitutionForm, self).__init__(*args, **kwargs)
-
-        self.fields['first_name'].required = False
-        self.fields['last_name'].required = False
-        self.fields['address'].required = False
-        self.fields['postal_code'].required = False
-        self.fields['email_address'].required = False
-        self.fields['banding'].required = True
+        super().__init__(*args, **kwargs)
+        self.fields['country'].required = True
+        self.fields['currency'].required = True
+        self.fields['size'].required = True
+        self.fields['level'].required = True
+        self.fields['fee'].required = False
+        self.fields['fee'].disabled = True
+
+    def save(self, commit=True):
```

Comment:
Can you add a quick explaination of the save methods behavior? It could be considered a little confusing that the save() method might discard the input and return a different object.